internal/manager/setup.go

```diff
@@ -318,11 +317,12 @@ func (c *Config) adminAPIClientFromServiceDiscovery(ctx context.Context, httpcli
 		return nil
 	},
 		retry.Context(ctx),
-		retry.Attempts(60),
-		retry.DelayType(retry.FixedDelay),
+		retry.Attempts(0),
+		retry.DelayType(retry.BackOffDelay),
 		retry.Delay(time.Second),
-		retry.OnRetry(func(_ uint, err error) {
-			logrus.New().WithError(err).Error("failed to create kong client(s)")
+		retry.MaxDelay(2*time.Minute),
```

Comment:
2 minutes might be a long time. Imaging that someone scales up from 0 and then needs to wait 2 minutes for KIC to pick this Admin API up.

I'd suggest to leave it at 1 second because of this and not log when we receive the error about not having endpoints. We can extract that error to a separate type of a variable and then match it in the `OnRetry()` below to get rid of the noise when we don't get any endpoints. WDYT?

Reply:
updated as your suggestion.